src/DatePicker.purs

```diff
@@ -609,7 +609,10 @@ renderItem index item =
       ]
     )
     -- printDay will format our item correctly
-    [ HH.text $ printDay item ]
+    [ HH.span
+      [ css (getLabelStyles item) ]
```

Comment:
I find it confusing that we have both `getLabelStyles` and `getSelectableStyles`. Can we not just use `getSelectableStyles` for both? Or can `getSelectableStyles` at least be the one to use `getLabelStyles`?

Reply:
There are two sets of style rules due to limitation of "strike-through" class name.

```css
.strike-through {
	position: relative;
}

.strike-through::before {
	position: absolute;
	content: "";
	left: 0;
	top: 50%;
	right: 0;
	border-top: 1px solid;
	border-color: inherit;
	-webkit-transform: rotate(-5deg);
	-moz-transform: rotate(-5deg);
	-ms-transform: rotate(-5deg);
	-o-transform: rotate(-5deg);
	transform: rotate(-5deg);
}
```

The tilted line is actually the upper border of the `::before` pseudo element so its color is controlled by `border-color` (`border-color: inherit`) not text `color`. But in `getCalendarStyles` we already have rules for border (`border-white` and `hover:border-blue-88`) so I have to apply the `strike-through` on a separate element.